docs/tutorial.rst

```diff
@@ -118,129 +118,118 @@ The Python interface
+4. Set the model as the generic piecewise constant size model, using the
+predefined human effective population size (see :ref:`sec_catalog`).
+Since we are providing one effective population size, the model is constant
+population size for one population over time.
 
+.. code-block:: python
 
+    >>> model = stdpopsim.PiecewiseConstantSize(species.population_size)
 
-.. _sec_tutorial_generic_models:
+5. Set the number of samples and set the simulation engine.
+In this case we will simulate 10 samples and use the default simulator,
+which is `msprime`. But, you can go crazy with the sample size!
+`msprime` is great at simulating large samples!
 
-**************
-Generic models
-**************
+.. code-block:: python
 
-The previous sections showed how to simulate population genetic models that have
-been published for particular species. It is also sometimes useful to simulate
-more generic models. We can still use the information for particular species to
-do this; the only difference is that we instantiate the models directly rather
-than retrieving them from the :ref:`catalog <sec_catalog>`.
+    >>> samples = model.get_samples(10)
+    >>> engine = stdpopsim.get_default_engine()
 
+6. Simulate the model with the contig length and number of samples we defined above.
+We capture the simulation results in a `tree sequence object <https://tskit.readthedocs.io/en/stable/python-api.html#tskit.TreeSequence>`_.
 
 .. code-block:: python
 
-    species = stdpopsim.get_species("homsap")
-    contig = species.get_contig("chr22", length_multiplier=0.1)
-    model = stdpopsim.PiecewiseConstantSize(species.population_size)
-    samples = model.get_samples(10)
-    engine = stdpopsim.get_default_engine()
-    ts = engine.simulate(model, contig, samples)
+    >> ts = engine.simulate(model, contig, samples)
 
+7. We can now do some simple checks that our simulation worked with ``tskit``.
 
-Here, we simulate 10% of human chromosome 22 under a constant size
-population model, using the current best estimate of the human
-effective population size from the **TODO add crossref to catalog section**
+.. code-block:: python
 
-.. :ref:`sec_catalog_homo_sapiens_genome`
+    >>> ts.num_samples
+    10
+    >>> ts.num_populations
+    1
+    >>> ts.num_mutations
+    6197
+    >>> ts.num_trees
+    6863
+
+As expected, there are 10 samples in one population. We can also see that 6197 mutations
+and 6863 trees were simulated (since we are not using a seed here, the number of mutations
+and trees will be slightly different for you). Try running the simulation again, and notice
+that the number of samples and populations stays the same, while the number of mutations
+and trees changes.
+
+8. In addition to working directly with the simulated tree squence, we can also output
+other common formats used for population genetics analyses.
+We can use ``tskit`` to convert the tree sequence to a vcf file called "foo.vcf".
+See the `tskit documentation
```

Comment:
Again, can link automatically.

Reply:
Haven't figured out how to get this to link.
I tried
```
:class:`tskit.TreeSequence.write_vcf`
```

Reply:
This is a method, so ``:meth:``

Reply:
thanks